anyway/flask_app.py

```diff
@@ -131,8 +132,8 @@ def generate_csv(results):
             output.writeheader()
 
         row = {k: v.encode('utf8')
-               if type(v) is six.text_type else v
-               for k, v in iteritems(serialized)}
+        if type(v) is str else v
```

Comment:
wrong indentation - this is part of the dict comprehension. Keep the indentation as it is.